scripts/installCommonDeps.sh

```diff
@@ -369,7 +369,7 @@ install_libsrtp2(){
 }
 
 install_node() {
-  local NODE_VERSION="v10.21.0"
+  local NODE_VERSION="v14.18.1"
```

Comment:
Can we just assign it to `14`? So it will use the latest release of version 14, which includes the latest fixes.